src/fontra/views/editor/editor.js

```diff
@@ -990,6 +999,10 @@ export class EditorController {
 
   async doubleClickedAnchorsCallback(event) {
     const glyphController = await this.sceneModel.getSelectedStaticGlyphController();
+    if (!glyphController.canEdit) {
+      this.sceneController._dispatchEvent("glyphEditLocationNotAtSource");
+      return;
+    }
```

Comment:
Suggest to add blank line after this, to be consistent with the similar block in doubleClickedGuidelinesCallback.

Reply:
done